checks.d/powerdns_authoritative.py

```diff
@@ -0,0 +1,69 @@
+# stdlib
+from collections import namedtuple
+
+# Datadog
+from checks import AgentCheck
+
+# 3p
+import requests
+
+class PowerDNSAuthoritativeCheck(AgentCheck):
+    # See https://doc.powerdns.com/md/authoritative/performance/#performance-monitoring for metrics explanation
+
+    SERVICE_CHECK_NAME = 'powerdns.authoritative.can_connect'
+
+    def check(self, instance):
+        config, tags = self._get_config(instance)
+        stats = self._get_pdns_stats(config)
+        for stat in stats:
+            self.log.debug('powerdns.authoritative.{}:{}'.format(stat['name'], stat['value']))
```

Comment:
Can you remove this logging statement? While debug logging is always more verbose, I don't think we should be logging everything that's collected from this check.